### ISSUE TITLE
[Feature Request]: Support for route loading in route group closure

### ISSUE THREAD

**Issue by ralphjsmit** [laravel-idea/plugin#718]
### Feature Description

We have the following code in our RouteServiceProvider:

```php
Route::middleware(['web', 'auth', 'email-verified', 'phone-verified', 'has-required-user-type', 'has-required-user-data', 'host'])
    ->prefix('host')
    ->name('host.')
    ->group(function () {
        // Route autocompletion provided
        require base_path('routes/host.php');
   
        // No route name autocompletion
        Route::middleware(['can:accessSomeService'])
            ->prefix('some-service')
            ->name('some-service.')
            ->group(base_path('routes/services/host/some-service.php'));

        // No route name autocompletion
        Route::middleware(['can:accessSomeOtherService'])
            ->prefix('some-other-service')
            ->name('some-other-service.')
            ->group(base_path('routes/services/host/some-other-service.php'));
    });
```

Since it is a rather big project, it would be great if autocompletion could be added for the two route files for which we don't have autocompletion yet. 

Thanks!

**Comment by adelf** [comment]
This will be fixed with big "route system" refactoring, which I hope will happen in March. Thank you for additional feature test :)

**Comment by ralphjsmit** [comment]
Thank you! Looking forward to it already. 👀

**Comment by adelf** [comment]
It's fixed in 7.0. I hope :)

**Comment by ralphjsmit** [comment]
@adelf Wow, this works FANTASTIC. This is so amazing, super job! ❤️
